### ISSUE TITLE
 got an unexpected keyword argument 'inherit_logging_config'

### ISSUE THREAD

**Issue by candrews67** [meraki/dashboard-api-ansible#66]
When I run ansible-playbook -i hosts who_am_i.yml  I get this output.  What am I doing wrong?  I do not see anything in the playbook about logging.

ansible-playbook -i hosts  who_am_i.yml 
[WARNING]: Collection cisco.meraki does not support Ansible version 2.12.10
[WARNING]: running playbook inside collection cisco.meraki
Operations to perform:
  Apply all migrations: admin, api, auth, contenttypes, db, sessions
Running migrations:
  No migrations to apply.

PLAY [meraki_servers] ********************************************************************************************************************************************
2024-10-11 14:16:23,192 INFO ansible: PLAY [meraki_servers] ********************************************************************************************************************************************

TASK [Get my administered identities] ****************************************************************************************************************************
2024-10-11 14:16:23,377 INFO ansible: TASK [Get my administered identities] ****************************************************************************************************************************
Friday 11 October 2024  14:16:23 -0500 (0:00:00.975)       0:00:00.975 ******** 
2024-10-11 14:16:23,394 INFO ansible: Friday 11 October 2024  14:16:23 -0500 (0:00:00.975)       0:00:00.975 ******** 
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: TypeError: __init__() got an unexpected keyword argument 'inherit_logging_config'
2024-10-11 14:16:23,425 ERROR ansible: An exception occurred during task execution. To see the full traceback, use -vvv. The error was: TypeError: __init__() got an unexpected keyword argument 'inherit_logging_config'
fatal: [127.0.0.1]: FAILED! => {"msg": "Unexpected failure during module execution.", "stdout": ""}
2024-10-11 14:16:23,426 ERROR ansible: fatal: [127.0.0.1]: FAILED! => {"msg": "Unexpected failure during module execution.", "stdout": ""}

PLAY RECAP *******************************************************************************************************************************************************
2024-10-11 14:16:23,457 INFO ansible: PLAY RECAP *******************************************************************************************************************************************************
127.0.0.1                  : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
2024-10-11 14:16:23,457 INFO ansible: 127.0.0.1                  : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   

Friday 11 October 2024  14:16:23 -0500 (0:00:00.206)       0:00:01.182 ******** 
2024-10-11 14:16:23,600 INFO ansible: Friday 11 October 2024  14:16:23 -0500 (0:00:00.206)       0:00:01.182 ******** 
=============================================================================== 
2024-10-11 14:16:23,601 INFO ansible: =============================================================================== 
Get my administered identities ---------------------------------------------------------------------------------------------------------------------------- 0.21s
2024-10-11 14:16:23,601 INFO ansible: Get my administered identities ---------------------------------------------------------------------------------------------------------------------------- 0.21s

**Comment by candrews67** [comment]
I got it to work on another server.I installed a new virtual machine. Ubuntu 22.04 and installed everything there and it works so I think I need to upgrade my current server (Ubuntu 20.04) and fully patch and try again.  That is my plan for tomorrow. I have high hopes that will resolve the problemOn Oct 15, 2024 1:14 PM, Francisco Muñoz Miranda ***@***.***> wrote:
Hi, @candrews67 could I see your playbook and your collection version?
